SourceGitphp/SourceGitphp.php

```diff
@@ -0,0 +1,367 @@
+<?php
+
+# Copyright (c) 2012 John Reese
+# Licensed under the MIT license
+
+if ( false === include_once( config_get( 'plugin_path' ) . 'Source/MantisSourcePlugin.class.php' ) ) {
+	return;
+}
+
+require_once( config_get( 'core_path' ) . 'url_api.php' );
+
+class SourceGitphpPlugin extends MantisSourcePlugin {
+	public function register() {
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
+
+		$this->version = '0.18';
+		$this->requires = array(
+			'MantisCore' => '1.2.0',
```

Comment:
*master* branch is for Mantis 1.3 so this dependency is incorrect, it will not be possible to install your plugin.